src/resource-scheduling/simulator.md

```diff
@@ -0,0 +1,89 @@
+# Simulator
+
+## Overview
+
+With its flexibility and significant benefits of reducing time and cost, the
+simulator plays an important role in studying and designing a computer
+architecture. It is often used to validate specific design schemes
+and evaluate the effectiveness of design schemes.
+
+## Workflow
+
+Since our targeted audience are people who are interested in TiKV, we will
```

Comment:
"We'll focus on how the simulator is used in TiKV to deal with scheduling problems"

Readers don't need to be told what they are.